Sources/WalletConnect.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Copyright © 2019 Gnosis Ltd. All rights reserved.
+//
+
+public class WalletConnect {
```

Comment:
Not sure if this class is actually needed - I have a feeling that it should be merged with the `Communicator` and both `Client` and `Server` inherit from the `Communicator`